src/MongoDB/Server.c

```diff
@@ -91,19 +88,18 @@ PHP_METHOD(Server, executeQuery)
 	char                     *namespace;
 	phongo_zpp_char_len       namespace_len;
 	zval                     *zquery;
-	zval                     *readPreference = NULL;
+	zval                     *zreadPreference = NULL;
```

Comment:
I'd probably have removed the "z" in front of query as well. Our coding standard doesn't generally do Hungarian Notation :-)